src/feedback.tsx

```diff
@@ -90,31 +79,28 @@ const useStyles = createUseStyles((theme) => ({
         background: ({isInverse}) => (isInverse ? theme.colors.backgroundBrand : theme.colors.background),
     },
 
-    '@keyframes sweepIn': {
-        '0%': {
-            opacity: 0,
-            transform: 'translate(0, 20px)',
-        },
-        '100%': {
-            opacity: 1,
-            transform: 'translate(0, 0)',
-        },
-    },
-
     innerContainer: {
         textAlign: 'left',
         padding: '64px 8px 0',
     },
 
     feedbackData: {
-        animation: animateText(theme.platformOverrides),
-        opacity: initialTextOpacity(theme.platformOverrides),
```

Comment:
This change is related to https://github.com/Telefonica/webapp/pull/907. When animations/transitions are disabled by CSS, the feedback text keeps `opacity: 0, translate(0,20px)`. We now manage the transition from React in order to be sure that the final state will always be reached.